todo_rxbloc_app/lib/feature_homepage/bloc/navigation_bloc.dart

```diff
@@ -1,19 +1,19 @@
 import 'package:rx_bloc/rx_bloc.dart';
 import 'package:rxdart/rxdart.dart';
-
+import 'package:todos_repository_core/todos_repository_core.dart';
 import '../../app_extensions.dart';
 import '../../base/enums/current_page_enum.dart';
+import '../../base/models/navigation_parametars.dart';
 import '../../base/routers/router.dart';
 
 part 'navigation_bloc.rxb.g.dart';
 
 abstract class NavigationBlocEvents {
-  @RxBlocEvent(type: RxBlocEventType.behaviour, seed: NavigationEnum.todosList)
-  void setPageIndex(NavigationEnum currentPage);
+  void navigate(NavigationParametars navigationParametars);
 }
 
 abstract class NavigationBlocStates {
-  Stream<NavigationEnum> get getPageIndex;
+  Stream<NavigationParametars> get getPageIndex;
```

Comment:
Please fix the typo `Parametars` -> `Parameters`. Also you can use the short option `NavigationParams `